test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -1542,7 +1554,7 @@ TEST_F(AtenXlaTensorTest, TestPow) {
   });
 }
 
-TEST_F(AtenXlaTensorTest, TestFmod) {
+TEST_F(AtenXlaTensorTest, TestFmodScalar) {
```

Comment:
We actually did not have fmod() on scalar, thanks for catching this.
Adding it now.